lib/timeline/TimeStep.js

```diff
@@ -579,12 +579,14 @@ TimeStep.prototype.getLabelMinor = function(date) {
   // noinspection FallThroughInSwitchStatementJS
   switch (this.scale) {
     case 'week':
-      if(this.isMajor() && date.weekday() !== 0){
-          return "";
+      if (this.isMajor() && date.weekday() !== 0) {
+        return "";
+      } else {
+        //default
+        return format && format.length > 0 ? this.moment(date).format(format) : '';  
```

Comment:
Oh ya, that sounds great to me ( I also prefer what I originally had, but didn't know I could add an ignore).